lib/callAll.js

```diff
@@ -463,24 +463,23 @@ module.exports = function(self, options) {
       return;
     }
     const isModified = Promise.promisify(self.isModified);
-    return isModified(req, doc).then(function(modified) {
-      // If there is no modification and that's not news, no update.
-      // Otherwise always update so we get the last editor's name
-      if ((modified === doc.workflowModified) && (!modified)) {
-        return;
-      }
-      const $set = {
-        workflowModified: modified
-      };
-      if (req.user && req.user._id && req.user.title) {
-        $set.workflowLastEditor = req.user.title;
-        $set.workflowLastEditorId = req.user._id;
-      }
-      return self.apos.docs.db.update({
-        _id: doc._id
-      }, {
-        $set: $set
-      });
+    const modified = await isModified(req, doc);
```

Comment:
not using try/catch in purpose so an error would be displayed in the console?

Reply:
It's being awaited further up by the promise event delivery mechanism, which will correctly handle it from there, because it is an `async` function. Whereas previously the use of `.then()` with no `.catch()` meant any rejection was just lost.